### PR TITLE
Add Nano support

### LINKED ISSUE #100: Support Arduino Nano
Firmware companion to https://github.com/MobiFlight/MobiFlight-Connector/issues/579.

This will require:

* A `board.h` file based off the Uno board
* A new entry in `platformio.ini` to generate Nano builds

No change to the GitHub build process is required, it was designed from the start to expect more Arduino boards in the future.

### PR DESCRIPTION
**Pull request by neilenns**
## Description of changes

Fixes #100 

* Add nano to `platformio.ini`
* Add `MFBoards.h` for the nano device

The board file came from @elral.

### CODE REVIEW

**Comment by neilenns** [comment]
@elral should review this to make sure I grabbed the correct `MFBoards.h` file for the nano.

File: platformio.ini [hunk 1]
```diff
@@ -100,3 +100,21 @@ lib_deps =
...
+framework = arduino
+build_flags = 
+	${env.build_flags}
+	-I./_Boards/Atmel/Board_Nano
+build_src_filter = 
+	${env.build_src_filter}
+	+<../_Boards/Atmel>
+lib_deps = 
+	${env.lib_deps}
+	arduino-libraries/Servo @ 1.1.8
```
Comment (elral) [inline platformio.ini:117]: The Servo library is already listed in the standard environment `[env]` and shouldn't be listed here.
Once the Raspeberry Pico (or other HW) gets supported the library must indeed moved to this location as the lib depends on the architecture.

**Review by elral — COMMENTED** [review]
The MFBoards.h file is fine.
But I have one small comment to the platformio.ini file (see below)

**Review by DocMoebiuz — APPROVED** [review]
Thanks @neilenns

**Pull request merged**